<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Настройка ссылок и путей</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(54, 59, 89);</string>
  </property>
  <widget class="QTableWidget" name="urlpathTable">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>481</width>
     <height>311</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #8388a4; /* Цвет фона */
border-color: rgb(131, 136, 164);
border-radius: 10px;
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Изменение путей и ссылок&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="safeBTN">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #8388a4; /* Цвет фона */
border: 1px solid #8388a4; /* Убираем границу кнопки */
border-radius: 20px;
padding: 0px;</string>
   </property>
   <property name="text">
    <string>СОХРАНИТЬ ИЗМЕНЕНИЯ</string>
   </property>
  </widget>
  <widget class="QLabel" name="infoLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
